statsmodels/stats/gof.py

```diff
@@ -144,12 +145,9 @@ def powerdiscrepancy(observed, expected, lambd=0.0, axis=0, ddof=0):
             if axis == 0:
                 e = e.T
 
-    if np.allclose(np.sum(e, axis=axis), n, rtol=1e-8, atol=0):
-        p = e/(1.0*nt)
-    elif np.allclose(np.sum(e, axis=axis), 1, rtol=1e-8, atol=0):
-        p = e
```

Comment:
Instead of removing `p`, it should be included in the return after changing return to HolderTuple